src/components/lifetime/celebration/singles-day/PlatformSection.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+const PlatformSection = ({ textContent }) => {
+  return (
+    <section className="overflow-hidden">
+      <div className="flex flex-col items-center py-12">
+        <div className="center flex items-center px-20 pb-16 text-center">
+          <p className="text-5xl font-semibold text-gray-100">{textContent.PlatformSection.title}</p>
+        </div>
+        <div className="center flex items-center px-20 pb-16 text-center">
+          <p className="text-xl font-semibold text-gray-80">{textContent.PlatformSection.subtitle}</p>
+        </div>
+        <div className="sm:gap-x-30 flex flex-row flex-wrap justify-center gap-y-10 gap-x-20 lg:gap-x-40">
+            <div className="flex flex-col items-center space-y-6">
```

Comment:
Here , you can do something like:
In fact, you can create an array of objects like:

````

const platforms = [
{
title: textContent.PlatformSection.web,
image: getImage('web_image_path'),
alt: 'Web image'
},
{
title: textContent.PlatformSection.iOS,
image: getImage('ios_image_path'),
alt: 'iOS icon'
},
...
];

platforms.map((platform) => (
<div className="flex flex-col items-center space-y-6">
                <div className="flex h-14 w-14 items-center justify-center rounded-full bg-gray-100">
                <Image src={platform.image} width={26.5} height={32} alt={platform.alt} className="filter invert" />
                </div>
                <p className="text-gray-80">{platform.title}</p>
            </div>
))
`````